cmd/api-linter/integration_test.go

```diff
@@ -23,27 +23,31 @@ import (
 	"testing"
 )
 
-var protocPath = func() string {
-	return "protoc"
-}
-
 // Each rule must have a testing case stored here.
```

Comment:
Remove this line of comment, as we don't require that each rule must have a testing case here. Thanks!

Reply:
Done.